spyderlib/config.py

```diff
@@ -530,6 +557,7 @@ def is_ubuntu():
               'editor/open file': "Ctrl+O",
               'editor/save file': "Ctrl+S",
               'editor/save all': "Ctrl+Shift+S",
+              'editor/save as': 'Meta+S',

```

Comment:
The standard is `Ctrl+Shift+S`, but it's used for something like panes, isn't it ?